### DISCUSSION TITLE
Slate and Geeko don't work

### DISCUSSION THREAD

**Discussion by cborne** [FreshRSS/FreshRSS#6810] [Q&A] [answered]
### Describe the bug

Hi, 
https://www.slate.fr/rss.xml
and https://geeko.lesoir.be/feed dont work since few day. 
Try in another feed reader like liferea and it works.

in logs : 
⚠️		2024-09-14 14:54:03 	cURL error 22: The requested URL returned error: 403 [https://www.slate.fr/rss.xml]
⚠️		2024-09-14 14:44:23 	cURL error 22: The requested URL returned error: 403 [https://geeko.lesoir.be/feed] 

### FreshRSS version

1.24.3 

### Environment information

 Votre version de PHP est la 8.3.11, qui est compatible avec FreshRSS.

Vous disposez de la librairie cURL.

Vous disposez de PDO et d’au moins un des drivers supportés (pdo_mysql, pdo_sqlite, pdo_pgsql).

Vous disposez de la librairie pour les expressions régulières (PCRE).

Vous disposez de la librairie pour la vérification des types de caractères (ctype).

Vous disposez de la librairie fileinfo.

Vous disposez de la librairie pour parcourir le DOM.

Vous disposez de l’extension recommandée JSON.

Vouz disposez de la librairie recommandée mbstring pour Unicode.

Vous disposez de l’extension ZIP.

**Comment by Alkarex** [comment, marked as answer]
* https://www.slate.fr/rss.xml is behind Cloudflare, which is breaking many RSS feeds. Try to go through https://feedburner.google.com
* https://geeko.lesoir.be/feed has a user-agent detection. Try to set `curl` in the user-agent field in FreshRSS when adding the feed

**Comment by nlegaillart** [comment]
Hi,

Do you know why Cloudflare breaks RSS feeds? Is it a side effect?

Feedburner feeds are not that easy to find without a Google account, I eventually managed to find one for slate.fr but not for korii.slate.fr :sob:

**Reply by Alkarex** [reply]
> Do you know why Cloudflare breaks RSS feeds? Is it a side effect?

Yes, they are knowingly breaking the Web. For RSS feeds, their introduction of JavaScript checks is stupid.
It is possible though for Web server admins to set their feeds up correctly, but this requires that they know and care.
It is always good to try to reach out to the Webmasters though.

> Feedburner feeds are not that easy to find without a Google account

With a Google account, you can just make your own feed to bypass Cloudflare checks.
This of course leads to more Internet centralisation.

**Reply by nlegaillart** [reply]
OK @Alkarex, thanks for your quick answer :+1:

**Comment by cborne** [comment]
Solved for me.

**Comment by Draky50110** [comment]
I think Slate had a tech problem solved now as feed is updated correctly.

**Reply by Alkarex** [reply]
The problem with Cloudflare is that it may randomly work for one client but not for another, based on e.g. IP, TLS settings, etc.
